src/tests/profiling/profile_kast_json.py

```diff
@@ -12,10 +14,18 @@
     from pyk.testing import Profiler
 
 
-def test_kast_json(profile: Profiler) -> None:
-    json_file = TEST_DATA_DIR / 'kast.json'
-    with json_file.open() as f:
-        json_data = json.load(f)
+KAST_JSON_TEST_DATA: list[tuple[str, str]] = [
+    ('kast-term', 'kast.json'),
+    ('compiled-defn', 'compiled.json'),
+]
+
+
+@pytest.mark.parametrize('test_id,file_name', KAST_JSON_TEST_DATA, ids=[test_id for test_id, *_ in KAST_JSON_TEST_DATA])
+def test_kast_json(profile: Profiler, test_id: str, file_name: str) -> None:
+    json_file = TEST_DATA_DIR / file_name
+    with profile('json-load.prof', sort_keys=('cumtime',), limit=20):
+        with json_file.open() as f:
+            json_data = json.load(f)
```

Comment:
Maybe we should split this up further like

```suggestion
    json_file = TEST_DATA_DIR / file_name
    json_text = json_file.read_text()

    with profile('json-load.prof', sort_keys=('cumtime',), limit=20):
        json_data = json.loads(json_text)
```

Reply:
Done.